site/assets/scss/_toc.scss

```diff
@@ -20,11 +22,16 @@
     ul {
       padding-left: 0;
       margin-bottom: 0;
+      margin-left: -.75rem;
       list-style: none;
 
       ul {
         padding-left: 1rem;
-        margin-top: .25rem;
+        margin-left: -1rem;
```

Comment:
Changed the behavior in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1837/commits/432d6a1cff8e22e77801c57a305efbdc7545e309 because I was trying to stick with the current commit from Bootstrap where the behavior was different compared to the _main_ branch.